source/guide_up1.rst

```diff
@@ -0,0 +1,217 @@
+.. author:: ezra <ezra@posteo.de>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/up1.png
+      :align: center
+
+###
+Up1
+###
+
+Up1_ provides a very simple web interface to share files end-to-end encrypted via an autogenerated url. It works without any registration or login, the encryption takes place in the clients browser, so the server will know nothing about the files content (see an example installation on on https://share.riseup.net).
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Node.js_ and its package manager npm_
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [isabell@stardust ~]$
+ 
+We will need to update the Node Packet Manager npm:
+
+::
+
+  [isabell@stardust ~]$ npm install npm@latest -g
+  + npm@6.4.1
+  updated 1 package in 14.629s
+  [isabell@stardust ~]$ 
+
+
+.. include:: includes/my-print-defaults.rst
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+First get the Up1 source code from Github_:
+
+::
+
+  [isabell@stardust ~]$ git clone https://github.com/Upload/Up1 up1 --depth 1
+  Cloning into 'up1'...
+  remote: Enumerating objects: 37, done.
+  remote: Counting objects: 100% (37/37), done.
+  remote: Compressing objects: 100% (33/33), done.
+  remote: Total 37 (delta 0), reused 16 (delta 0), pack-reused 0
+  Unpacking objects: 100% (37/37), done.
+  Note: checking out '059f199a6f94fc6c90cde1e4dab0b5b85b49533c'.
+  [isabell@stardust ~]$
+
+You have now downloaded the master development branch, because this might behave like a beta version of the software, you should check out the latest release. Please be sure to replace the pseudo version number ``66.6`` here with the latest version number from the release feed_:
+
+::
+
+  [isabell@stardust ~]$ cd ~/up1
+  [isabell@stardust up1]$ git checkout v66.6 --depth1
```

Comment:
You probably meant `git checkout tags/v1.0`, but that won't work either if repo was cloned with `--depth 1` (see above)

Reply:
ah yes correct, thats the result of a dirty change in the clone/checkout process :P 
Earlier I did the checkout already together with the clone command, but thought its easier to understand when I  split these two actions. But then it wont work with `clone --depth 1`. But `checkout v66.6` actually works, git can handle this. But of course its better to be verbose in the guide, so I will adapt this.